packages/amplify_core/lib/src/category/amplify_auth_category.dart

```diff
@@ -1223,6 +1295,37 @@ class AuthCategory extends AmplifyCategory<AuthPluginInterface> {
             ),
           );
 
+  /// {@template amplify_core.amplify_auth_category.set_up_totp}
+  /// Initiates setup of a time-based one-time passcode (TOTP) MFA method for the
+  /// current user.
+  ///
+  /// **NOTE**: Your backend must be set up with TOTP MFA enabled prior to calling this method.
+  ///
+  /// The [TotpSetupDetails] returned contains a [TotpSetupDetails.getSetupUri]
+  /// method which can be used to display a QR code or render a button to open
+  /// the user's installed authenticator app. After setup is complete on the user's
+  /// end, call [verifyTotpSetup] with a one-time code to complete registration.
```

Comment:
I might reword this section to make it clear `TotpSetupDetails.getSetupUri` return's the URI. Right now it reads like it shows the relevant QR code. 

```suggestion
  /// The [TotpSetupDetails] returned contains a [TotpSetupDetails.getSetupUri]
  /// method for retrieving the setup URI. Which can be used to build a QR code or 
  /// a button to open the user's installed authenticator app. After setup is complete
  ///  on the user's end, call [verifyTotpSetup] with a one-time code to complete registration.
```

Reply:
Done 👍